Sources/Widgets/Widgets3D/LineWidget/index.js

```diff
@@ -174,12 +184,15 @@ function vtkLineWidget(publicAPI, model) {
   });
 
   // Default manipulator
-  model.manipulator = vtkPlanePointManipulator.newInstance();
+  publicAPI.setManipulator(
+    vtkPlanePointManipulator.newInstance({ useCameraNormal: true })
+  );
```

Comment:
a manipulator could be set via initial values, hence you should not overwrite it